dark/intervals.py

```diff
@@ -87,6 +88,20 @@ def coverage(self):
                 coverage += (min(end, self._targetLength) - max(0, start))
         return float(coverage) / self._targetLength
 
+    def baseCoverage(self):
+        """
+        For each base in the subject, return a count of how many times that
+        base is covered by a read.
+
+        @return: a C{Counter} where the keys are the positions on the subject
+            and the value the number of time that position is covered by a
+            read.
+        """
+        baseCoverages = Counter()
+        for interval in self._intervals:
+            baseCoverages.update(list(range(interval[0], interval[1])))
```

Comment:
You don't need the `list` here. The update function iterates over its argument. Even simpler is just

```python
baseCoverages.update(range(*interval))
```